### PR TITLE
Separate news directory for each plugin

### PR DESCRIPTION
**Pull request by shagunsodhani**
## Motivation

Separate news directory for each plugin

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebookresearch/hydra/blob/master/CONTRIBUTING.md)?

Yes

## Test Plan

CI

## Related Issues and PRs

(Is this PR part of a group of changes? Link the other relevant PRs and Issues here. Use https://help.github.com/en/articles/closing-issues-using-keywords for help on GitHub syntax)

#1177

### CODE REVIEW

File: plugins/hydra_ax_sweeper/news/_template.rst [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{% for section in sections %}
```
Comment (jieru-hu) [inline plugins/hydra_ax_sweeper/news/_template.rst:1]: Sorry for the noob question. would it be possible to share the `_template.rst` some way?

Comment (omry) [reply plugins/hydra_ax_sweeper/news/_template.rst:1]: Good question.
Looking at the code of towncrier, it looks like it's possible to specify the template file to use in pyproject.toml.

File: plugins/hydra_colorlog/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+[tool.towncrier]
```
Comment (omry) [inline plugins/hydra_colorlog/pyproject.toml:1]: I think you should keep:
feature, bugfix, config and docs.

Comment (shagunsodhani) [reply plugins/hydra_colorlog/pyproject.toml:1]: We have had some instances of other types in Ax plugin. For example:

1. api_change -> https://github.com/facebookresearch/hydra/issues/913
2. maintenance -> https://github.com/facebookresearch/hydra/pull/782/files

File: plugins/hydra_ax_sweeper/pyproject.toml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+  [[tool.towncrier.type]]
+  directory = "trivial"
+  name = "Trivial Changes"
+  showcontent = false
+
+  [[tool.towncrier.type]]
+  directory = "maintenance"
+  name = "Maintenance Changes"
+  showcontent = true
```
Comment (omry) [inline plugins/hydra_ax_sweeper/pyproject.toml:44]: You missed my request to keep only:
feature, bugfix, api_change and config.

Do this for all plugins.

Comment (shagunsodhani) [reply plugins/hydra_ax_sweeper/pyproject.toml:44]: tagged you on the previous comment

File: plugins/hydra_colorlog/pyproject.toml (hunk 2 above)
Comment (shagunsodhani) [reply plugins/hydra_colorlog/pyproject.toml:1]: @omry ^^

Comment (shagunsodhani) [reply plugins/hydra_colorlog/pyproject.toml:1]: Doesnt seem obvious. Will look around more

Comment (shagunsodhani) [reply plugins/hydra_colorlog/pyproject.toml:1]: Does not seem straightforward. Should we merge this, as is?

Comment (omry) [reply plugins/hydra_colorlog/pyproject.toml:1]: Yes

**Review by omry — APPROVED** [review]

**Pull request merged**